### DISCUSSION TITLE
see the edit button in a view

### DISCUSSION THREAD

**Discussion by jordanvax** [filamentphp/filament#14951] [Help]
### Package

Other

### Package Version

v3.2.124

### How can we help you?

I have a table that manages relationships. At the moment I have a route that manages the detail and a view that manages the edit, the current table is visible in both pages except that in the edit route the modify button is visible. I want to move from this management to a detail page where I can edit the modal life directly without going through an edit route. I have a problem with the behaviour of the relationship manager. I can't force the modify button to be visible in the detail view. Here is the current component: 
class ServicesRelationManager extends RelationManager
{
    protected static string $relationship = 'services';

    public static function getTitle(Model $ownerRecord, string $pageClass): string
    {
        return __('gnss_receiver');
    }

    protected function canEdit(Model $record): bool
{
    return true;
}

    public function form(Form $form): Form
    {
        return $form
            ->columns(1)
            ->schema([
                Forms\Components\ToggleButtons::make('active')
                    ->label(__('box'))
                    ->boolean()
                    ->grouped(),

                Forms\Components\DatePicker::make('end_subscription')
                    ->label(__('End subscription'))
                    ->required(),
            ]);
    }

    public function table(Table $table): Table
    {
        return $table
            ->recordTitleAttribute('gnss_receiver_id')
            ->columns([
                Tables\Columns\TextColumn::make('projectTypes.name')
                    ->formatStateUsing(fn (string $state): string => __($state)),

                Tables\Columns\TextColumn::make('active')
                    ->label(__('Status'))
                    ->formatStateUsing(fn (int $state) => $state ? __('Active') : __('out_of_service'))
                    ->badge()
                    ->color(fn (int $state): string => $state ? 'success' : 'danger')
                    ->icon(fn (int $state): string => $state ? 'heroicon-m-check' : 'heroicon-m-exclamation-triangle'),

                Tables\Columns\TextColumn::make('end_subscription')
                    ->label(__('End warranty'))
                    ->date(config('constants.date_time_format.fr_reduced'))
                    ->formatStateUsing(fn (Model $record, string $state): View => view(
                        'filament.infolists.components.service-entry-content',
                        ['active' => $record->active, 'endSubscription' => $state],
                    )),
            ])
            ->actions([
                Tables\Actions\EditAction::make(),
            ]);
    }
}

**Comment by leandrocfe** [comment]
I didin't understand your question.. could you show some screenshots that you are trying to achieve please?

**Reply by jordanvax** [reply]
thank you for your reply, I'll try to be more precise, on the first photo it's a table that manages a relationship. On the second photo it's the same component but it's visible when you want to modify a resource. I can't figure out how to display the modify button. 
<img width="1215" alt="Capture d’écran 2024-12-01 à 12 04 06" src="https://github.com/user-attachments/assets/553d652b-7748-4173-b8d3-fe74e106a8c3">
<img width="1220" alt="Capture d’écran 2024-12-01 à 12 09 07" src="https://github.com/user-attachments/assets/5e6e2918-40a1-403c-b8cf-3993c2890e18">
